parts/linux/cloud-init/nodecustomdata.yml

```diff
@@ -300,16 +308,15 @@ write_files:
     runtime-endpoint: unix:///run/containerd/containerd.sock
     #EOF
 
-{{if IsKubenet }}
 - path: /etc/sysctl.d/11-containerd.conf
```

Comment:
Do you need this conf file all the time? or just it's containerd?

Reply:
I believe moby-engine automatically sets these ip_forwarding configs so not needed for docker. The new containerd VHDs won't have moby-engine at all so we need to set it manually.

Reply:
nvm I see it's already inside {{if NeedsContainerd}}